vagrant/provisioning/roles/alfresco/tasks/main.yml

```diff
@@ -1,6 +1,14 @@
 - include_tasks: alfresco_user_and_folders.yml
 - include_tasks: alfresco_downloads.yml
 
+- name: Alfresco packages
+  become: yes
```

Comment:
libreoffice support per https://docs.alfresco.com/6.1/tasks/lo-install.html and https://docs.alfresco.com/5.1/concepts/OOoJodconverter-subsystem-props.html